website/content/v1.1/talos-guides/upgrading-talos.md

```diff
@@ -20,7 +20,9 @@ This will simply update the boot reference and reboot.
 Unless explicitly told to `preserve` data, an upgrade will cause the node to wipe the [EPHEMERAL]({{< relref "../learn-more/architecture/#file-system-partitions" >}}) partition, remove itself from the etcd cluster (if it is a control node), and generally make itself as pristine as is possible.
 (This is generally the desired behavior, except in specialised use cases such as single-node clusters.)
 
-*Note* that unless the Kubernetes version has been specified in the machine config, an upgrade of the Talos Linux OS will also apply an upgrade of the Kubernetes version.
+*Note* that the default since Talos version 1.0 is to specify the Kubernetes version in the machine config.
```

Comment:
shouldn't this also go in the 1.0 docs?

Reply:
Good catch, I'd missed it off the commit :+1: